.mailmap

```diff
@@ -1459,7 +1459,7 @@ kangzhiq <709563092@qq.com> kkkkx <709563092@qq.com>
 klaasvanaarsen <44929042+klaasvanaarsen@users.noreply.github.com>
 lastcodestanding <rohang71604@gmail.com>
 latot <felipematas@yahoo.com>
-luzpaz <luzpaz@users.noreply.github.com> luz.paz <luzpaz@users.noreply.github.com>
+luzpaz <luzpaz@users.noreply.github.com> luz paz <luzpaz@pm.me>
```

Comment:
I think probably both these lines are needed.

Reply:
I'll fix it so it reads `luz.paz`

Reply:
Done in [108affd](https://github.com/sympy/sympy/pull/24125/commits/108affd9a807261179165f3e3860ba210c8e4998)

Reply:
You can run this authors check yourself locally:
```console
$ bin/mailmap_check.py 
Ambiguous names in .mailmap
This email address appears for multiple entries:
Person: luz.paz <luzpaz@users.noreply.github.com>
Mailmap entries:
luzpaz <luzpaz@users.noreply.github.com> luz.paz <luzpaz@pm.me>
This author is not included in the .mailmap file:
luz paz <luzpaz@pm.me>

The .mailmap file needs to be updated because there are commits with
unrecognised author/email metadata.

Lines should be added to .mailmap to indicate the correct name and
email aliases for all commits.

For instructions on updating the .mailmap file see:
https://github.com/sympy/sympy/wiki/Development-workflow#add-your-name-and-email-address-to-the-mailmap-file
```

Reply:
Addressed in [f371962](https://github.com/sympy/sympy/pull/24125/commits/f371962568f29d798ce9757d8e1430fd54f59c47)